src/test/csf/Peer.h

```diff
@@ -145,9 +145,9 @@ struct Peer
         }
 
         boost::optional<Ledger>
-        acquire(Ledger::ID const & id)
+        acquire(Ledger::ID const & lid)
```

Comment:
consider `lID`

Reply:
Fixed in 42d4b481a `Misc fixups in shadow warnings commit`